stytch/consumer/cryptowallets/types.go

```diff
@@ -0,0 +1,110 @@
+	SessionDurationMinutes int32 `json:"session_duration_minutes,omitempty"`
+	// SessionJWT: The `session_jwt` associated with a User's existing Session.
+	SessionJWT string `json:"session_jwt,omitempty"`
+	// SessionCustomClaims: Add a custom claims map to the Session being authenticated. Claims are only created
+	// if a Session is initialized by providing a value in `session_duration_minutes`. Claims will be included
+	// on the Session object and in the JWT. To update a key in an existing Session, supply a new value. To
+	// delete a key, supply a null value.
+	//
+	//   Custom claims made with reserved claims ("iss", "sub", "aud", "exp", "nbf", "iat", "jti") will be
+	// ignored. Total custom claims size cannot exceed four kilobytes.
+	SessionCustomClaims map[string]any `json:"session_custom_claims,omitempty"`
+}
+
+// AuthenticateStartParams: Request type for `CryptoWallets.AuthenticateStart`.
+type AuthenticateStartParams struct {
+	// CryptoWalletType: The type of wallet to authenticate. Currently `ethereum` and `solana` are supported.
+	// Wallets for any EVM-compatible chains (such as Polygon or BSC) are also supported and are grouped under
+	// the `ethereum` type.
+	CryptoWalletType string `json:"crypto_wallet_type,omitempty"`
+	// CryptoWalletAddress: The crypto wallet address to authenticate.
+	CryptoWalletAddress string `json:"crypto_wallet_address,omitempty"`
+	// UserID: The unique ID of a specific User.
+	UserID string `json:"user_id,omitempty"`
+	// SessionToken: The `session_token` associated with a User's existing Session.
+	SessionToken string `json:"session_token,omitempty"`
+	// SessionJWT: The `session_jwt` associated with a User's existing Session.
+	SessionJWT string `json:"session_jwt,omitempty"`
+}
+
+// AuthenticateResponse: Response type for `CryptoWallets.Authenticate`.
+type AuthenticateResponse struct {
+	// RequestID: Globally unique UUID that is returned with every API call. This value is important to log for
+	// debugging purposes; we may ask for this value to help identify a specific API call when helping you
+	// debug an issue.
+	RequestID string `json:"request_id,omitempty"`
+	// UserID: The unique ID of the affected User.
+	UserID string `json:"user_id,omitempty"`
+	// SessionToken: A secret token for a given Stytch Session.
+	SessionToken string `json:"session_token,omitempty"`
+	// SessionJWT: The JSON Web Token (JWT) for a given Stytch Session.
+	SessionJWT string `json:"session_jwt,omitempty"`
+	// User: The `user` object affected by this API call. See the
+	// [Get user endpoint](https://stytch.com/docs/api/get-user) for complete response field details.
+	User users.User `json:"user,omitempty"`
+	// StatusCode: The HTTP status code of the response. Stytch follows standard HTTP response status code
+	// patterns, e.g. 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX
+	// are server errors.
+	StatusCode int32 `json:"status_code,omitempty"`
```

Comment:
Just noting that the old type is `int`, but since it's the status code the change should be fine